taxonomies/allergens.txt

```diff
@@ -10,6 +10,7 @@ stopwords:pt:traços,vestígios,vestígio,a,e,uma,em,ou,de,que,com,contém,cont
 
 en:none, allergens-free, without allergens, without, n, no, no allergens, 0
 fr:aucun, aucune, sans allergènes, sans, pas, pas d'allergènes, aucun allergène, 0
+it:no allergeni
```

Comment:
We need complete translations, in order to match what we can get in the Excel files from producers.

```suggestion
it:nessuno, nessuna, senza allergeni, no allergeni, no, n, 0
```

Reply:
@teolemon : why did you mark this resolved?

Reply:
I tried to commit it actually

Reply:
and it's done, so closing now